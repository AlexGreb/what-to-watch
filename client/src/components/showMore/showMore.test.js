/**
 * @jest-environment jsdom
 */
import React from 'react';
import {render} from '@testing-library/react';
import ShowMore from './showMore.jsx';

describe(`ShowMore`, () => {
  it(`Should render correctly a ShowMore component`, () => {
    const {asFragment} = render(<ShowMore/>);
    expect(asFragment()).toMatchSnapshot();
  });
});
