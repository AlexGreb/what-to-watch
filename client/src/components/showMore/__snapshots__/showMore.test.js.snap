// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMore Should render correctly a ShowMore component 1`] = `
<DocumentFragment>
  <button
    class="catalog__button"
    type="button"
  >
    Show more
  </button>
</DocumentFragment>
`;
